src/lib/__tests__/Preview-test.js

```diff
@@ -1002,6 +1002,15 @@ describe('lib/Preview', () => {
             expect(preview.retryTimeout).to.equal(undefined);
         });
 
+        it('should load preview when a well-formed file object is passed and server update should be skipped', () => {
+            const previewOptions = { skipServerUpdate: true };
+            preview.parseOptions(previewOptions);
```

Comment:
Since we are unit testing ```load```, you shouldn't call parseOptions. Instead set the required fields on the preview object

Reply:
Thank you, done.